src/Components/Components/src/Forms/FieldChangedEventArgs.cs

```diff
@@ -1,21 +1,27 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System;
+
 namespace Microsoft.AspNetCore.Components.Forms
 {
     /// <summary>
     /// Provides information about the <see cref="EditContext.OnFieldChanged"/> event.
     /// </summary>
-    public sealed class FieldChangedEventArgs
+    public sealed class FieldChangedEventArgs : EventArgs
     {
         /// <summary>
-        /// Identifies the field whose value has changed.
+        /// Creates a new instance of <see cref="FieldChangedEventArgs"/>.
         /// </summary>
-        public FieldIdentifier FieldIdentifier { get; }
-
-        internal FieldChangedEventArgs(in FieldIdentifier fieldIdentifier)
+        /// <param name="fieldIdentifier">The <see cref="Forms.FieldIdentifier"/></param>
+        public FieldChangedEventArgs(in FieldIdentifier fieldIdentifier)
```

Comment:
I made these three bois constructable. There's really no good reason why not, and without it, it would be hard for someone to unit test code that uses these types.